spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/pubsub/it/PubSubChannelAdaptersIntegrationTests.java

```diff
@@ -204,28 +200,20 @@ public MessageChannel outputChannel() {
 			return new QueueChannel();
 		}
 
-		@Bean
-		public GcpProjectIdProvider gcpProjectIdProvider() {
-			return () -> "bliss";
-		}
-
-		@Bean
-		public CredentialsProvider credentialsProvider() {
-			return new NoCredentialsProvider();
-		}
-
 		@Bean
 		public SubscriberFactory defaultSubscriberFactory(
 				@Qualifier("subscriberExecutorProvider") ExecutorProvider executorProvider,
 				TransportChannelProvider transportChannelProvider,
-				PubSubAdmin pubSubAdmin) {
+				PubSubAdmin pubSubAdmin,
+				GcpProjectIdProvider projectIdProvider,
+				CredentialsProvider credentialsProvider) {
 			if (pubSubAdmin.getSubscription("doralice") == null) {
```

Comment:
Could it become a problem with these constant names like we did with the pubsub IT tests when several were running (from multiple PRs). Perhaps append a random bit at the end? (topic and subscription)